src/styles/block/page.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  &__cards {
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    width: $container-width;
+    padding: 50px $container-padding;
+  }
```

Comment:
Add justify-content: center; to center cards in container even if we delete one of them.

Reply:
do we really need to keep cards in the center after deletion?

Reply:
No, we don't. You are right, Tetiana !